### PR TITLE
Allow support for this.owner

### PR DESCRIPTION
**Pull request by rwwagner90**
On the latest ember-qunit, I am getting `No container found to lookup service from!` This is because `this.container` and `this.application` are now removed. The new thing to lookup from is `this.owner`, which I have added here.

### CODE REVIEW

**Comment by simonihmig** [comment]
@rwwagner90 Thank you, looks good! 

I am just wondering, can we cover this with a test? I guess if we add another integration test file with the new QUnit API, with a test like https://github.com/kaliber5/ember-window-mock/blob/master/tests/integration/window-test.js#L14-L27, this would hopefully be failing without your changes here?

**Comment by rwwagner90** [comment]
@simonihmig I added a test, works like a charm 👍 . Good to merge now?

**Comment by rwwagner90** [comment]
I'm not sure why tests are failing on Travis. Tried updating to latest Ember to see if it will help.

**Comment by rwwagner90** [comment]
Seems to have fixed things 👍

File: tests/test-helper.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import Application from '../app';
+import config from '../config/environment';
 import { setApplication } from '@ember/test-helpers';
 import { start } from 'ember-qunit';
 
-setApplication(Application.create({ autoboot: false }));
+setApplication(Application.create(config.APP));
```
Comment (simonihmig) [inline tests/test-helper.js]: I think this was causing the failed tests, IIRC some ember-cli 2.17 blueprint files were not working correctly with the new testing API. Had this in another repo, upgrading to ember-cli 2.18 fixed it (with `ember-cli-update`, so updated blueprint files would also be applied). But hey, why not go directly to 3.0? ;)

Comment (rwwagner90) [reply tests/test-helper.js]: Might as well 😄

**Comment by rwwagner90** [comment]
Would you be able to add me as a collaborator on npm and/or release a new version?

**Comment by simonihmig** [comment]
> Would you be able to add me as a collaborator on npm and/or release a new version?

Added you! Will release once #39 is merged, wanted to have this fixed...

**Comment by simonihmig** [comment]
Released as 0.2.1

**Pull request merged**